doc/source/NewSiteConfigs.rst

```diff
@@ -13,11 +13,13 @@ The instructions below are for GNU (`gcc`), since this is the easiest and best s
 +-------------------------------------------+----------------------------------------------------------------------+---------------------------+
 | Compiler                                  | Versions tested/in use in one or more site configs                   | Spack compiler identifier |
 +===========================================+======================================================================+===========================+
-| Intel classic (icc, icpc, ifort)          | 2021.3.0 to the final version in oneAPI 2023.2.4 [#fn1]_             | ``intel@``                |
+| Intel classic (icc, icpc, ifort)          | 2021.3.0 to the final version in oneAPI 2023.2.4 (2021.10.0) [#fn1]_ | ``intel@``                |
 +-------------------------------------------+----------------------------------------------------------------------+---------------------------+
-| Intel mixed (icx, icpx, ifort)            | 2024.1.2 to 2024.2.0                                                 | ``oneapi@``               |
+| Intel mixed (icx, icpx, ifort)            | 2024.1.2 to 2025.0.0                                                 | ``oneapi@``               |
 +-------------------------------------------+----------------------------------------------------------------------+---------------------------+
-| GNU (gcc, g++, gfortran)                  | 9.2.0 to 13.3.0 (note: 14.x.y is **not** yet supported)              | ``gcc@``                  |
+| Intel LLVM (icx, icpx, ifx)               | 2024.2.1 to 2025.0.0                                                 | ``oneapi@``               |
++-------------------------------------------+----------------------------------------------------------------------+---------------------------+
+| GNU (gcc, g++, gfortran)                  | 9.2.0 to 13.3.1 (note: 14.x.y is **not** yet supported)              | ``gcc@``                  |
```

Comment:
If gcc 13 is now supported, should the `brew install` "pin" at line 143-144 be updated to v13?

Reply:
So far, macOS users have used the gcc@12 gfortran together with apple-clang. I don't expect any problems with gcc@13, but I haven't tried (and I no longer use macOS). If somebody wants to try and report back, then I am happy to update, otherwise leave as is (gcc@12 is perfectly fine for all our use cases).